client/src/components/AddNewProfile.js

```diff
@@ -16,21 +17,41 @@ const AddNewProfile = () => {
     support_type: "",
     profile_type: "",
     status: "new",
+    nationality: "",
     join_date: new Date()
   });
-
+  const [errors, setErrors] = useState([]);
   const [profileCreated, setProfileCreated] = useState(null);
+  const [nationalities, setNationalities] = useState([]);
 
   const handleChange = event => {
     updateField(event.target.name, event.target.value);
   };
 
   const createProfile = () => {
+    let errs = [];
+    if (profileData.firstname.length === 0) {
+      errs.push(<li><p>First name cannot be empty</p></li>);
+    }
+    if (profileData.lastname.length === 0) {
+      errs.push(<li><p>Last name cannot be empty</p></li>);
+    }
+    if (profileData.phone.length < 10) {
+      errs.push(<li><p>Phone number must be 10 characters</p></li>);
```

Comment:
I would make this "Phone number must have at least 10 digits" since most are 11 or more.

Also, make sure that we allow empty values for fields (like phone) which can be null in the database. These should be sent as `null` in the JSON request rather than empty strings.